notebooks/hive2bq/HiveToBigquery_notebook.ipynb

```diff
@@ -86,23 +89,24 @@
    "execution_count": null,
    "id": "b3742b9a-143d-49fc-b43c-e56179c7f0f2",
    "metadata": {
-    "tags": [],
     "pycharm": {
      "name": "#%%\n"
-    }
+    },
+    "tags": []
    },
    "outputs": [],
    "source": [
     "# Google Cloud notebooks requires dependencies to be installed with '--user'\n",
     "! pip3 install pyspark\n",
     "! pip3 install --upgrade google-cloud-pipeline-components kfp --user -q\n",
```

Comment:
I think there is a discussion on the OracleToBigQuery PR that is relevant here too so I'll hold off on looking at this until the other one is resolved.